### DISCUSSION TITLE
Vercel dashboard - new project dashboard not loading GitHub repos

### DISCUSSION THREAD

**Discussion by aberba** [vercel/community#1765] [Help]
### Summary

For some reason, our Vercel dashboard doesn't show the list of repos from our GitHub account when we try to create a new project. It show a "loading" state but gets stuck at loading. 

![image](https://user-images.githubusercontent.com/8314648/224722061-475a4c41-508b-4e7e-bfa9-65209e36d760.png)

We have tried reconnecting the repos and even allowing access to all our repos. Yet it doesn't see to sold the "loading" issue. 

Is anyone experiencing such an issue? What could be the cause?

**Comment by amyegan** [comment]
Hi @aberba. I was not able to reproduce this problem on my own account. Is it still not working for you?

**Comment by aberba** [comment]
@amyegan sorry, no more with my old company. I believe it was resolved.
